content/learn/userguides/secondary-analysis/10x.md

```diff
@@ -36,10 +39,32 @@ Cell barcode correction uses as input files in BAM file format; thus the fastq f
+
+Docker: [Python3-scientific](https://github.com/HumanCellAtlas/skylab/blob/master/docker/python3-scientific/Dockerfile)
+
+Key Library: [Sctools](https://github.com/HumanCellAtlas/sctools)
+
 ## Alignment
 
-The [STAR](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/) alignment software is then used to map the reads to the genome, using the GRCh38 human genome primary sequence as reference. STAR (Spliced Transcripts Alignment to a Reference) is widely-used for RNA-seq alignment and generates transcript count information as well as gene identification assignments. Alignment to the V27 GenCode transcriptomic reference is also performed using the [HISAT2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4655817/) software to ensure that actual biotypes (and not pseudogenes, for example) are identified.
+The [STAR](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/) alignment software is used to map the reads to the GRCh38 human genome primary assembly reference. STAR (Spliced Transcripts Alignment to a Reference) is widely-used for RNA-seq alignment and identifies the best matching location(s) on the reference for each sequencing read.
+
+## Gene Annotation
+
+The [TagGeneExon]() tool from [DropSeqTools] is used to annotate each read with the type of sequence it aligned to. These annotations include `INTERGENIC`, `INTRONIC`, and `EXONIC`, and are stored in the `XF` tag. In cases where the gene corresponds to an intron or exon, the name of the gene that overlaps the alignment is associated with the read and stored in the `GE` tag.
+
+## Duplicate Marking
+
+Optical and PCR duplicates are marked using [`UmiAwareMarkDuplicates`]() tool. This tool corrects groups reads based on their UMI, gene, and alignment position. Future improvements will make this tool group by cell barcode as well [See this picard issue](). In groups containing more than one sequencing read, one read is selected as primary, and the rest are marked as duplicates. This tool also corrects cell barcodes using an approach modified from [Jaitin et al.](), wherein reads are subsumed into larger groups when they share the same position and gene, and the UMI is within an edit distance of 1.
 
 ## Metrics
 
 A number of quality control tools are used to assess the quality of the data output each time this pipeline is run. For a list of the tools and information about each one please see [this page] of our documentation.
+
+## Count Matrix Construction
+
+The pipeline outputs a count matrix that contains, for each cell and for each gene, the number of molecules that were observed. In practice, this equates to counting up the number of sequencing reads with 255 map quality (uniquely aligned to the genome) that are not duplicates (are the representative member of one or more alignments that share the same UMI, and therefore derive from the same molecule). The count matrix is then generated as a tab-separated
```

Comment:
I'm assuming you meant to add the word 'file' at the end; is that the end of the sentence or is there more to say about the count matrix?